packages/rebound/test_rebound.py

```diff
@@ -0,0 +1,20 @@
+from pyodide_test_runner import run_in_pyodide
+
+
+@run_in_pyodide(
+    packages=[
+        "rebound",
```

Comment:
You have to add `numpy` here as in
```suggestion
        "rebound","numpy",
```
```